### ISSUE TITLE
Carbon footprint informations

### ISSUE THREAD

**Issue by fch25804** [ovh/public-cloud-roadmap#181]
As a public cloud user, I think it may be interesting to have information about the carbon footprint of the resources I use to seek for ways to reduce it.
It may be available for each instance, summed up by project, and also in a global view. With as many details as possible.
Our CSR committee regularly ask for this type of data.

**Comment by JacquesMrz** [comment]
@fch25804 , this has started on our side. Public Cloud might not be the first product universe which will display those information (spoiler, it's BareMetal) but it's definitely on its way.
I'll keep you updated as soon as we have information to share.

**Comment by lombre33** [comment]
This will be a game changer ! At Komet Xperience we are waiting this kind of tools with a lot of excitement.

**Comment by baaastijn** [comment]
Hello,

First step is carbon footprint this summer for dedicated servers. For vm/db
and co it's more complex due to multi tenancy

We detailed this topic during Very Tech Trip, I recommend the replay here :
https://player.vimeo.com/video/804448757?

**Comment by biapar** [comment]
Any news?

**Comment by yomovh** [comment]
Hello,
You should be able to see the carbon footprint entry when you click on your account or directly through this [link](https://www.ovh.com/manager/#/carbon-calculator/dashboard/)
As said by @baaastijn  this contains only the dedicated servers information. We are working on the integration of public cloud data.

**Comment by drzraf** [comment]
While on it, if OVH allowed customers to rely on actual requested resources (network / disk / RAM / CPU) by offering the ability to scale **down** (#71, like switching **back** and forth between B/C/D/R instances, even **below** B2-flex), that would be even more _carbon-efficient_ 😉

**Comment by daboooo** [comment]
Hello,

Any news?

I read in this [article](https://blog.ovhcloud.com/carbon-impact-of-cloud-services-dont-get-fooled/) that you plan to extend the tool to the Public Cloud for full coverage of your range of cloud services in 2024. Is this still on track for this year?

Also, I believe this collaborative tool could be relevant for OVH: [Datavista](https://dataviz.boavizta.org/cloudimpact).
It seems it would be easy for you to contribute by adding OVH services (AWS services are already available).

Regards

**Comment by DavidDelebecque** [comment]
Hi,

Thanks for your inputs.
Carbon footprint for Public cloud IaaS (Compute/Storage) should be available End of 1st Quarter of 2025.
You will be able to download the carbon footprint report from your previous month. An API will also be available to doawnlaod them (with a 24 months rolling history).

Hope it answers your questions,

**Comment by DavidDelebecque** [comment]
Hi,

Carbon footprint for Compute/Instances will be available in March for February Public Cloud Compute consumption.
https://help.ovhcloud.com/csm/en-carbon-footprint-calculator?id=kb_article_view&sysparm_article=KB0066739

Stay tuned !

**Comment by DavidDelebecque** [comment]
Carbon footprint live for Compute instances !
Storage coming soon

**Comment by biapar** [comment]
Where? I looked into my account and I don't see it.

**Comment by DavidDelebecque** [comment]
Here, you can find the way to go
https://help.ovhcloud.com/csm/en-carbon-footprint-calculator?id=kb_article_view&sysparm_article=KB0066739

**Comment by fch25804** [comment]
Any way to enable the feature for admin contact?

**Comment by DavidDelebecque** [comment]
> Any way to enable the feature for admin contact?

The calculation is based on the services billed. 
By extension, we make the calculation those emissions available for NIC billing

**Comment by snk33** [comment]
@DavidDelebecque I can see some very big changes between late 2024 and 2025 values for HGR servers and Hosted Private Cloud Hosts (both vSAN and standard PRE).
Is there any method changes made recently on calculations ? (to prepare PCI addition ?)

**Comment by DavidDelebecque** [comment]
> [@DavidDelebecque](https://github.com/DavidDelebecque) I can see some very big changes between late 2024 and 2025 values for HGR servers and Hosted Private Cloud Hosts (both vSAN and standard PRE). Is there any method changes made recently on calculations ? (to prepare PCI addition ?)

The "method" did not change. However, changes have been added in some transformation factors to estimate greenhouse gases for example. For transparency, we will soon publish all details in our corporate website.

**Comment by snk33** [comment]
> > [@DavidDelebecque](https://github.com/DavidDelebecque) I can see some very big changes between late 2024 and 2025 values for HGR servers and Hosted Private Cloud Hosts (both vSAN and standard PRE). Is there any method changes made recently on calculations ? (to prepare PCI addition ?)
> 
> The "method" did not change. However, changes have been added in some transformation factors to estimate greenhouse gases for example. For transparency, we will soon publish all details in our corporate website.

That's weird, for example with manufacturing/electricity/operations between december 2024 and february 2025 :
- HGR-STOR server @ RBX : 335 / 78 / 4 = 417 -> 209 / 15 / 7 = 231
- PRE384 vSAN @ RBX : 24 / 33 / 4 = 61 -> 12 / 20 / 7 = 39

I feel like HGR-STOR values were somehow wrong...

Curious to get methodology details !

**Comment by daboooo** [comment]
> Storage coming soon

@DavidDelebecque is it on the road ? 🙂

**Comment by DavidDelebecque** [comment]
> > Storage coming soon
> 
> [@DavidDelebecque](https://github.com/DavidDelebecque) is it on the road ? 🙂

Yes, it is clearly on the road. We had to deal with different others priorities but clearly scheduled for end of 2025. Hopefully Sept/Oct..
I let you consult this FR more related to this topic : https://github.com/ovh/public-cloud-roadmap/issues/797

**Comment by daboooo** [comment]
Hello,

For several days, I have seen quite a few articles about a new OVH tool called "Impact Tracker."
I have therefore carefully read the official press release, which indeed announces the launch of this new tool on July 8th: [OVHcloud Press Release](https://corporate.ovhcloud.com/fr/newsroom/news/environmental-impact-tracker/).

However, I cannot find anything new in the OVH Manager.
Could you please help me find where this new tool is located?

Have a nice day.

**Comment by DavidDelebecque** [comment]
Hi,
Thabks for your input.
We are globally improving scope and accuracy : "OVHcloud is continuing to improve its Environmental Impact Tracker by integrating new cutting-edge functionalities. The latest innovations now make it possible to take into account emissions from the usage phase, with two angles, using both the average emission factors of the countries and those of the electricity suppliers with which the Group works, as well as the reconditioning of components during the server assembly phase. These improvements provide OVHcloud customers a better understanding of the carbon footprint of their cloud infrastructure and encourage more sustainable practices."

For public cloud, the goal is to have full coverage of public cloud services for 2026.

**Comment by daboooo** [comment]
So, it's not actually a new tool, is it?

It's the same feature that allows you to download a PDF report each month, but more accurate, right?
Because this tool has been available since the end of February.

So, can you confirm that there is no new feature?

**Comment by DavidDelebecque** [comment]
> So, it's not actually a new tool, is it?
> 
> It's the same feature that allows you to download a PDF report each month, but more accurate, right? Because this tool has been available since the end of February.
> 
> So, can you confirm that there is no new feature?

To sum up, same tool and full coverage for public cloud services + multi-criteria (water + abiotic resources) in 2026
